pod-operation/src/components/lim_current.rs

```diff
@@ -0,0 +1,52 @@
+use ads1x1x::ic::{Ads1015, Resolution12Bit};
+use ads1x1x::interface::I2cInterface;
+use ads1x1x::mode::OneShot;
+use ads1x1x::ChannelSelection::{SingleA0, SingleA1, SingleA2};
+use ads1x1x::{Ads1x1x, SlaveAddr, FullScaleRange};
+use nb::block;
+use rppal::i2c::I2c;
+
+fn voltage_to_current(voltage: i16) -> f32 {
+
+    let SLOPE = 15.873;
+    let YINTERCEPT = 39.683;
+
+    // Convert voltage to f32 and scale it
+    let voltage = f32::from(voltage) / 1000.0;
+    
+    // Multiply voltage by 15.873 and subtract 39.683
+    let current = (voltage * SLOPE - YINTERCEPT) as f32;
+
+    // Print the voltage value
+    println!("Voltage: {}", voltage);
+    
+    // Return the current value
+    current
```

Comment:
Chore: these comments are purely verbose and do not add any value, so these could be removed.